.github/workflows/release.yml

```diff
@@ -3,18 +3,21 @@ name: Update And Release
 on:
   schedule:
     - cron: '0 */6 * * *'
-  
+
+permissions:
+  contents: write
+
 jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: Use Node.js 14.x
-      uses: actions/setup-node@v2.1.5
+    - uses: actions/checkout@dc323e67f16fb5f7663d20ff7941f27f5809e9b6  # tag: v2
+    - name: Use Node.js 16
+      uses: actions/setup-node@1f8c6b94b26d0feae1e387ca63ccbdc44d27b561  # tag: v2
       with:
-        version: 14.x
+        node_version: 16
     - name: npm install and release
       run: npm run release
       env:
-        GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Maybe a dumb question, but do we know how this is actually named in our secrets? Just don't want to change the name if it's actually called `GH_TOKEN`

Reply:
Very valid question, and that part is a me stabbing a little blindly since I don't have perms to view the secrets. 🙂

GitHub actions provides `secrets.GITHUB_TOKEN` by default and scopes it based on the permissions listed in the workflow.

I think you're right, this change will probably cause breakage on releases - looks like it's used on [this line](https://github.com/electron/releases/blob/41534a4db5efd75a062cf73139ba904a2eaee979/script/release.sh#L8) to authenticate as `electron-bot`, so it can't just use `secrets.GITHUB_TOKEN`.

I believe `GH_TOKEN` got revoked as part of the security stuff, it [stopped working the other day](https://github.com/electron/releases/actions/runs/3849209771/jobs/6557937475).

I'll refactor the code a bit to make it more clear what that token is used for and to use it separately from `GITHUB_TOKEN`, which can be used for the Octokit usage.

Reply:
I've refactored the code to separate out `secrets.ELECTRON_BOT_TOKEN` - the secret will need to be renamed, but it needs to be rotated anyway since it was revoked (along with `secrets.NPM_AUTH_TOKEN` most likely).

Separating out the token for `electron-bot` should actually make this repo a bit safer since now the token being used for the Test workflow (that runs on PRs) only has read permissions.